modules/osd-aws-privatelink-firewall-prerequisites.adoc

```diff
@@ -72,11 +73,10 @@ This section provides the necessary details that enable you to control egress tr
 |443
 |Provides core container images as a fallback when quay.io is not available.
 |===
-+
-[NOTE]
-====
-Creating a firewall with a ROSA private cluster (non-PrivateLink) is not supported.
-====
+//[NOTE]
+//====
+//Creating a firewall with a ROSA private cluster (non-PrivateLink) is not supported.
+//====
 +
 When you add a site such as `quay.io` to your allowlist, do not add a wildcard entry such as `*.quay.io` to your denylist. In most cases, image registries use a content delivery network (CDN) to serve images. If a firewall blocks access, then image downloads are denied when the initial download request is redirected to a host name such as `cdn01.quay.io`.
 +
```

Comment:
For some reason, this `+` sign is visible in the Netlify preview. The line in the preview reads:

> "+ CDN host names, such as cdn01.quay.io, are covered when you add a wildcard entry, such as .quay.io, in your allowlist."

I created a local preview and the plus sign is not seen. I'm wondering if this was resolved after the latest auto-preview was built. I thought I'd mention it here just in case.